README.md

```diff
@@ -1,90 +1,99 @@
-UUID   Name
-
-========================================== User Services ==========================================
-UUID           Name         Ports                                 Status
-3ca8b4c1c834   redis        client: 6379/tcp -> 127.0.0.1:58508   RUNNING
-8a8c18860df1   voting-app   http: 80/tcp -> 127.0.0.1:58511       RUNNING
-```
+<img src="./readme-static-files/redis-voting-app-run-output.png">
+
+Finally, open the `http` link printed in the last line in your browser.
+
+If you have an issue or feature request, we'd love to hear about it through one of the following:
+
+- [Asking for help on our Discord server][discord]
+- Filing an issue on our [Github](https://github.com/kurtosis-tech/kurtosis/issues/new/choose) (which can also be done via `kurtosis feedback --bug` or `kurtosis feedback --feature`)
+- [Messaging us on Twitter][twitter]
+
+### Going further
+
+To try more Kurtosis packages just like this one, check out the [`awesome-kurtosis` repo][awesome-kurtosis] or one of these packages:
+
+- [Ethereum](https://github.com/kurtosis-tech/eth2-package): fully functional private Ethereum network in Kurtosis with Flashbots MEV-boost, any EL and CL client combination, and a collection of network monitoring tools.
+- [DIVE](https://github.com/HugoByte/DIVE): A CLI + Kurtosis package by [Hugobyte](https://hugobyte.com) for the ICON ecosystem that can spin up EVM, Cosmos, or JVM networks with a bridge between them.
+- [NEAR](https://github.com/kurtosis-tech/near-package): A private NEAR network in Kurtosis.
+
+To learn about how to write Kurtosis packages, check out our [quickstart][quickstart-reference].
+
+To read about how Kurtosis works, see [our documentation][docs].
 
-If this piqued your interest, you might like our [quickstart][quickstart-reference].
+To see where we're going with the product, check out the roadmap [here](https://github.com/kurtosis-tech/kurtosis/wiki/Short%E2%80%90term-Roadmap).
 
-### More Examples
+Why Kurtosis over Compose, Helm, Terraform, etc.?
+=================================================
+These tools have been around for over a decade, yet most developers still struggle to build distributed applications. Why? In a sentence: building distributed applications is hard, and these tools don't make it easy enough for the average developer.
 
-Further examples can be found in our [`awesome-kurtosis` repo][awesome-kurtosis].
+Some of our observations:
+
+- No tool works across the whole software lifecycle: Compose is good at quick local environments and bad at Prod environments, while Helm and Terraform are the opposite. This often means a dedicated DevOps team handles Prod deployment, leading to the same "throw it across the wall" problems the DevOps movement was founded around.
+- Compose, Helm, and Terraform all struggle to do sequential "first this, then this" logic necessary for many prototyping workflows.
```

Comment:
I would reframe this as well, just switching out struggle with "aren't designed to"